parfive/tests/localserver.py

```diff
@@ -56,7 +56,7 @@ def default_request_handler(self, environ, start_response):
         status = "200 OK"
         response_headers = [
             ("Content-type", "text/plain"),
-            ("Content-Disposition", "attachment; filename=testfile_{self.request_number}.txt"),
+            ("Content-Disposition", f"attachment; filename=testfile_{self.request_number}.txt"),
```

Comment:
I assume this was supposed to be an f-string originally? Also useful in the update test